repo/packages/E/elastic/200/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "packagingVersion": "4.0",
+  "upgradesFrom": [
+    "2.2.0-5.6.5"
+  ],
+  "downgradesTo": [
+    "2.2.0-5.6.5"
+  ],
+  "minDcosReleaseVersion": "1.10",
```

Comment:
minDcosReleaseVersion seems wrong here. It should be 1.9 as with other data services. @triclambert @benclarkwood